rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactory.groovy

```diff
@@ -26,7 +26,18 @@ class LocalJobFriendlyPackerCommandFactory implements PackerCommandFactory {
     def packerCommand = [baseCommand, "packer", "build", "-color=false"]
     parameterMap.each { key, value ->
       if (key && value) {
-        def keyValuePair = value instanceof String && value.contains(" ") ? "$key=\"$value\"" : "$key=$value"
+        def cleansedValue = value instanceof String ? value.trim() : value
```

Comment:
This change was due to the Packer issue I reported ([#4478](https://github.com/mitchellh/packer/issues/4478)). The issue arose when there were 2 or more packages in the packages variable that was being sent to packer. When building the packer command, the value of the "packages" parameter included quotes, which were then being included in the escaped quotes due to the space. This quotes within quotes was being included in the Powershell script which caused a parsing issue. The root issue might be with the way Apache Commons Exec handles quoted quotes (there's a related [issue](https://issues.apache.org/jira/browse/EXEC-54) open since 2012 that seems to relate), but this approach was easier. One of the packer contributors worked on this issue with me and confirmed that the way this now passes it's command line arguments to packer is okay even if the quotes are missing.